pyface/ui/qt4/code_editor/pygments_highlighter.py

```diff
@@ -21,6 +21,11 @@ def get_tokens_unprocessed(self, text, stack=('root',)):
     """ Split ``text`` into (tokentype, text) pairs.
 
         Monkeypatched to store the final stack on the object itself.
+
+        The `text` parameter this gets passed is only the current line, so to
```

Comment:
I'd prefer to leave it as is if that is fine, only to maintain exact match with the ipython code https://github.com/ipython/ipython/pull/6878/files .